src/Illuminate/Encryption/Encrypter.php

```diff
@@ -36,7 +36,7 @@ class Encrypter implements EncrypterContract, StringEncrypter
      *
      * @var array
      */
-    private static $supportedCiphers = [
+    public static $supportedCiphers = [
```

Comment:
Why do we need to change this?

Reply:
To be able to access it in the tests. If there is a better way to do it, please change it or let me know.

Reply:
Should just manually use a data provider in tests, making this public may cause 3rd party to override the default and we really don't want that

Reply:
It's this better now? Or would you do it differently? @crynobone https://github.com/laravel/framework/pull/53407/commits/ffee580f012e14152156710efbb4a5421059620a